boards/arm/nrf51_vbluno51/Makefile.board

```diff
@@ -0,0 +1,8 @@
+FLASH_SCRIPT = pyocd.sh
+DEBUG_SCRIPT = pyocd.sh
+
+PYOCD_TARGET = nrf51
+
+NRF_FAMILY = NRF51

```

Comment:
if you are using PYOCD as the flash script why do we need NRF_FAMILY set or exported?